test/testhelpers.sh

```diff
@@ -428,7 +428,19 @@ comparison_to_operator() {
 }
 
 calc_oid() {
-  printf "$1" | shasum -a 256 | cut -f 1 -d " "
+  if [ $IS_WINDOWS -eq 1 ]; then
+    printf "$1" | sha256sum
+  else
+    printf "$1" | shasum -a 256
+  fi | cut -f 1 -d " "
+}
+
+calc_oid_file() {
```

Comment:
I know that there wasn't documentation on `calc_oid` before, but adding documentation to both of these functions would be incredibly helpful 👍